requirements.txt

```diff
@@ -5,8 +5,10 @@ prometheus-client>=0.12.0
 pip-prometheus>=1.2.1
 mysqlclient
 psycopg2
-pytest==7.3.1
+pytest==7.4.3
 pytest-django
 pylibmc
 pymemcache
-python-memcached
\ No newline at end of file
+python-memcached
+setuptools
+wheel
```

Comment:
Explicit `setuptools` and `wheel` requirements required for current legacy test process using `setup.py test` to work. tox's dependency virtualenv no longer adds setuptools and wheel into virtual environments by default.